Realm.Shared/Handles/ObjectHandle.cs

```diff
@@ -127,8 +127,13 @@ private static class NativeMethods
 
             [DllImport(InteropConfig.DLL_NAME, EntryPoint = "object_remove_row", CallingConvention = CallingConvention.Cdecl)]
             public static extern void remove_row(ObjectHandle handle, out NativeException ex);
+
+            [DllImport(InteropConfig.DLL_NAME, EntryPoint = "object_equals_object", CallingConvention = CallingConvention.Cdecl)]
+            public static extern IntPtr equals_object(ObjectHandle handle, ObjectHandle otherHandle, out NativeException ex);
         }
 
+        public static readonly ObjectHandle Null = new ObjectHandle(null);
```

Comment:
This looks strange to me. Can you give me and @kristiandupont a bit more background?